draft-ietf-jsonpath-base.md

```diff
@@ -1066,9 +1066,11 @@ regex-expr   = regex-op regex
 regex-op     = "=~"                                   ; regular expression match
 regex        = <TO BE DEFINED>
 
-contain-expr = in-op container
+contain-expr = containable in-op container
```

Comment:
Would `content` would be a better term than `containable`?

Reply:
I consider `content` far too general. Perhaps `contain-val` or `contain-value` is a more specific and better term.

Reply:
I was aiming for symmetry with `container`. If we want to be more specific, then how about something like the following?
```suggestion
contain-expr = primitive-value in-op array-value
```